pkgs/desktops/pantheon/desktop/elementary-onboarding/default.nix

```diff
@@ -1,66 +1,50 @@
 { lib
 , stdenv
 , fetchFromGitHub
-, fetchpatch
 , nix-update-script
-, substituteAll
-, pkg-config
 , meson
 , ninja
-, vala
+, pkg-config
 , python3
-, gtk3
+, vala
+, wrapGAppsHook4
+, appcenter
+, elementary-settings-daemon
 , glib
-, granite
+, granite7
+, gtk4
+, libadwaita
 , libgee
-, elementary-settings-daemon
-, gettext
-, libhandy
-, wrapGAppsHook
-, appcenter
 }:
 
 stdenv.mkDerivation rec {
   pname = "elementary-onboarding";
-  version = "6.1.0";
+  version = "7.0.0";
 
   src = fetchFromGitHub {
     owner = "elementary";
     repo = "onboarding";
     rev = version;
-    sha256 = "sha256-9voy9eje3VlV4IMM664EyjKWTfSVogX5JoRCqhsUXTE=";
+    sha256 = "sha256-bxOy9VivpgL4xXJhDF7K/gpq9zcCFIJFfRpG7QC8svE=";
   };
 
-  patches = [
-    (substituteAll {
-      src = ./fix-paths.patch;
-      appcenter = appcenter;
-    })
```

Comment:
I later found that there is no need to patch this because it first checks whether appcenter is in $PATH and will not display AppCenterView when appcenter is not found. It make no sense to wrap appcenter since the app handle the case (without appcenter installed) well and appcenter is not part of the default Pantheon installation you will just get some useless options if you don't actually use appcenter.